prince-of-versions/CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+## Version 4.0.3
+
+_2021-04-02_
+
+- fix null value parsing
+    User should receive proper null values when certain fields (ie. version) are null
+
```

Comment:
Could you also document the change in parsing of metadata regarding null values?